build/images/base/Dockerfile

```diff
@@ -16,7 +16,7 @@ ARG BUILD_TAG
 FROM ubuntu:22.04 as cni-binaries
 
 ARG CNI_BINARIES_VERSION
-ARG WHEREABOUTS_VERSION=v0.5.4
```

Comment:
I searched WHEREABOUTS and found this `WHEREABOUTS_CNI_REL_TAG=v0.5.4`, you may also update this one.